### PR TITLE
Update to Elm 0.19, correct LICENSE spelling.

### PR DESCRIPTION
**Pull request by 4onen**
Saw issue #6 and couldn't help myself. I'm not absolutely one-hundred percent sure that I got every shiftLeftBy and shiftRightBy in the right places or with the right values, but it passes its tests and the example can successfully encode and decode other Base64 translators' values. As far as I know, that's good enough. 

(P.S: I have no idea how Elm package management works, so I edited the JSON to line up dependencies until it compiles. You may want to take a glance at those to make sure the package is still publishable.)

### CODE REVIEW

**Comment by justgook** [comment]
ok - ill pull and recheck it..

File: example/Translate.elm [hunk 1]
```diff
@@ -116,35 +117,47 @@ toStr r =
...
+        [] -> "[]"
+        [i] -> "["++String.fromInt i++"]"
+        ls -> "["++listIntToStringHelp ls++"]"
+
+listIntToStringHelp : List Int -> String
+listIntToStringHelp l =
+    case l of
+        [] -> "Damnit, how???"
+        i::[] -> String.fromInt i
+        i::is -> (String.fromInt i ++ ", ")++(listIntToStringHelp is)
```
Comment (justgook) [inline example/Translate.elm]: Maybe just apply elm-format to that file

File: src/BinaryBase64.elm [hunk 2]
```diff
@@ -55,7 +43,7 @@ type alias Octet =
 type alias ByteString =
     List Octet
 
-
+{-
```
Comment (justgook) [inline src/BinaryBase64.elm]: not comment out - just remove

**Comment by justgook** [comment]
looks like broke CI (Tavis build), can You or would like me to do that ?
looks like you only need to update version in `.travis.yml`

**Comment by 4onen** [comment]
Resolved those comments, but the CI build is still broken. The .travis.yml was trying to use elm-test instead of elm-test@beta, ELM_VERSION 0.18 instead of 0.19, and elm-make instead of elm make. I did my best, but I've no idea how to correct fetching the elm command as I've never worked within a docker instance before.

**Comment by 4onen** [comment]
Yeah, I could use a hand with this. I've no idea how to resolve the Travis errors.

**Comment by justgook** [comment]
Ok -  ill investigate..

**Comment by justgook** [comment]
Moved work to new one #8

**Pull request closed without merging**